src/themes/masschallenge/elements/label.overrides

```diff
@@ -16,6 +16,10 @@
   }
 }
 
+ui.label.tiny {
```

Comment:
Rename this as `tiny` already exists. You will override other places its used. `https://react.semantic-ui.com/elements/label/#variations-size`